### PR TITLE
cloud: support scaling up storage size

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/scale-tidb-cluster.md [hunk 1]
```diff
@@ -11,32 +11,37 @@ aliases: ['/tidbcloud/beta/scale-tidb-cluter']
...
+To scale a running cluster, take the following steps:
 
-1. Navigate to the TiDB Clusters page and click the name of a cluster that you want to scale. The overview page of the cluster is displayed.
+1. Navigate to **Active Clusters** page and click the name of a cluster that you want to scale. The overview page of the cluster is displayed.
 2. In the cluster information pane on the left, click **Setting**.
 3. Click **Scale** in the drop-down menu. The **Scale** window is displayed.
-4. In the **Scale** window, increase the node quantity of your TiDB, TiKV, and TiFlash<sup>beta</sup> respectively.
+4. In the **Scale** window, you can make the following changes:
+
+    - TiDB: increase or decrease the the node quantity.
```
Comment (qiancai) [inline tidb-cloud/scale-tidb-cluster.md:23]:
```suggestion
    - TiDB: increase or decrease the node quantity.
```

File: tidb-cloud/size-your-cluster.md [hunk 2]
```diff
@@ -67,7 +67,12 @@ Minimum number of TiKV nodes: `ceil(3584 ÷ 1024) × 3 = 12`
 
 ### TiKV storage size
 
-You can configure the TiKV storage size only when you create or restore a cluster.
+- 8 vCPU or 16 vCPU TiKV supports up to 4 TiB storage capacity
+- 4 vCPU TiKV supports up to 2 TiB storage capacity
```
Comment (qiancai) [inline tidb-cloud/size-your-cluster.md:71]:
```suggestion
- 8 vCPU or 16 vCPU TiKV supports up to 4 TiB storage capacity.
- 4 vCPU TiKV supports up to 2 TiB storage capacity.
```

File: tidb-cloud/release-notes-2022.md [hunk 3]
```diff
@@ -44,43 +48,43 @@ This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cl
...
 ## May 13, 2022
 
-* Support [signing up](https://tidbcloud.com/signup) TiDB Cloud with a Google account
+* Support [signing up](https://tidbcloud.com/signup) TiDB Cloud with a Google account.
 
 ## May 1, 2022
 
-* Support configuring vCPU size of TiDB, TiKV, and TiFlash<sup>beta</sup> when you create or restore a cluster
+* Support configuring vCPU size of TiDB, TiKV, and TiFlash<sup>beta</sup> when you create or restore a cluster.
 * Add the support of the AWS region `Mumbai` for cluster creation
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:68]:
```suggestion
* Add the support of the AWS region `Mumbai` for cluster creation.
```

File: tidb-cloud/scale-tidb-cluster.md [hunk 4]
```diff
@@ -11,65 +11,62 @@ aliases: ['/tidbcloud/beta/scale-tidb-cluter']
...
-5. Click **Confirm**.
+## Scale in a cluster
 
-## Scale in TiKV or TiFlash<sup>beta</sup> nodes
+To make the following scale-in or scale-down changes, you need to submit a support ticket. We will contact you and complete the scaling within the agreed time.
 
-To scale in TiKV nodes or TiFlash<sup>beta</sup> nodes, you need to submit a scale-in support ticket. We will contact you and complete the scale-in within the agreed time.
+- Decrease the node size of TiDB, TiKV, or TiFlash<sup>beta</sup>
+- Scale in TiKV or TiFlash<sup>beta</sup> nodes
+- Decrease the storage size of TiKV or TiFlash<sup>beta</sup>
```
Comment (DanielZhangQD) [inline tidb-cloud/scale-tidb-cluster.md:56]: This is not supported, only increasing the storage size is supported.

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**